src/ComponentSummary.jsx

```diff
@@ -269,34 +299,105 @@ return (
           },
           notifyAccountId: accountId,
           button: (starCount, starIsActive, starOnClick) => (
-            <Button type="button" onClick={starOnClick} aria-label="Star this component">
-              {starIsActive ? (
-                <i className="bi bi-star-fill" style={{ color: "var(--amber10)" }} />
-              ) : (
-                <i className="bi bi-star" />
-              )}{" "}
-              {starCount}
-            </Button>
+            <Widget
+              src="near/widget/DIG.Tooltip"
+              props={{
+                content: context.accountId
+                  ? starIsActive
+                    ? "Unstar this app"
+                    : "Star this app"
+                  : "Sign in to star this app",
+                trigger: (
+                  <Button type="button" onClick={starOnClick} aria-label="Star this component">
+                    {starIsActive ? (
+                      <i className="bi bi-star-fill" style={{ color: "var(--amber10)" }} />
+                    ) : (
+                      <i className="bi bi-star" />
+                    )}{" "}
+                    {starCount}
+                  </Button>
+                ),
+              }}
+            />
           ),
         }}
       />
 
-      <OverlayTrigger placement="top" overlay={<Tooltip>Copy URL to clipboard</Tooltip>}>
-        <Button
-          type="button"
-          onMouseLeave={() => {
-            State.update({ copiedShareUrl: false });
-          }}
-          onClick={() => {
-            clipboard.writeText(shareUrl).then(() => {
-              State.update({ copiedShareUrl: true });
-            });
+      {pinnedAppsFeatureEnabled() && (
+        <Widget
+          src="${REPL_ACCOUNT}/widget/SocialIndexActionButton"
+          key="social-index-action-pin"
+          props={{
+            actionName: "pin",
+            actionUndoName: "unpin",
+            item: {
+              type: "social",
+              path: src,
+            },
+            notifyAccountId: accountId,
+            onCommitStart: (pinIsActive) => {
+              emitPinnedAppsGatewayEvent(pinIsActive);
+            },
+            onCommitFailure: (pinIsActive) => {
+              emitPinnedAppsGatewayEvent(pinIsActive);
+            },
+            button: (pinCount, pinIsActive, pinOnClick) => (
+              <Widget
+                src="near/widget/DIG.Tooltip"
```

Comment:
```suggestion
                src="${REPL_ACCOUNT}/widget/DIG.Tooltip"
```

Reply:
@shelegdmitriy Whoops! I had a brain fart here. Thanks so much for catching that! I did a global find and replace and found a few other spots that needed to use `${REPL_ACCOUNT}`.